protocols/newFile/protos/newFile.proto

```diff
@@ -0,0 +1,52 @@
+syntax = "proto3";
+option go_package = "github.com/functionland/go-fula/protocols/newFile";
+package newFile;
```

Comment:
newFile seems like an odd package name; how about `file`?

Reply:
I will change the name to `file` and remove the old package after I finished the tests for the `mobile` package.